test/snippet/range/view/get.cpp

```diff
@@ -1,17 +1,23 @@
+#include <vector>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/alphabet/quality/aliases.hpp> // includes seqan3::dna4q
+#include <seqan3/alphabet/quality/phred42.hpp>
 #include <seqan3/core/debug_stream.hpp>
 #include <seqan3/range/view/get.hpp>
 #include <seqan3/range/view/to_char.hpp>
-#include <seqan3/alphabet/quality/aliases.hpp>
-
-using namespace seqan3;
 
 int main()
 {
-//! [usage]
-// Create a vector of dna4 quality composite alphabet.
-std::vector<dna4q> qv{{'A'_dna4, phred42{0}}, {'C'_dna4, phred42{1}}, {'G'_dna4, phred42{2}}, {'T'_dna4, phred42{3}}};
+    using seqan3::operator""_dna4;
+    using seqan3::operator""_phred42;
+
+    // Create a vector of dna4 quality composite alphabet.
+    std::vector<seqan3::dna4q> qv{{'A'_dna4, '0'_phred42},
+                                  {'C'_dna4, '1'_phred42},
+                                  {'G'_dna4, '2'_phred42},
+                                  {'T'_dna4, '3'_phred42}};
 
-debug_stream << (qv | view::get<0> | view::to_char) << std::endl; // Prints [A,C,G,T]
-debug_stream << (qv | view::get<1> | view::to_char) << std::endl; // Prints [!,",#,$]
-//! [usage]
+    seqan3::debug_stream << (qv | seqan3::view::get<0> | seqan3::view::to_char) << std::endl; // Prints [A,C,G,T]
+    seqan3::debug_stream << (qv | seqan3::view::get<1> | seqan3::view::to_char) << std::endl; // Prints [!,",#,$]
```

Comment:
```suggestion
    seqan3::debug_stream << (qv | seqan3::view::get<1> | seqan3::view::to_char) << '\n'; // Prints [!,",#,$]
```

Never use `std::endl`, always use `'\n'` (also in the line above).